pkgs/applications/misc/waybar/default.nix

```diff
@@ -65,28 +68,17 @@ stdenv.mkDerivation rec {
     )
   '';
 
-  nativeBuildInputs = [
-    meson ninja pkg-config scdoc wrapGAppsHook
-  ] ++ lib.optional withMediaPlayer gobject-introspection;
+  nativeBuildInputs = [ meson ninja pkg-config scdoc wrapGAppsHook systemd ]
```

Comment:
Why systemd in nativeBuildInputs?

Reply:
It's used in [meson.build](https://github.com/Alexays/Waybar/blob/effad1a5c3111fa8fecf44b6129772c7cd144990/meson.build#L147) to retrieve the user units directory to put the waybar.service file into.

Reply:
maybe we add a short comment here as this is a bid unusual

Reply:
It's only used at compile time, which from my understanding is what distinguishes nativeBuildInputs and buildInputs (also used at run time), but I may be wrong?

Reply:
Yes, nativeBuildInputs is for programs running on the build machine and buildInputs is for the target machine. Runtime dependencies are detected by nix.

Reply:
I'm not sure where to put it then, it's never actually run on any machine, only used by meson to get the pkg-config variable that I set [a few lines below](https://github.com/NixOS/nixpkgs/blob/3d27246376b85d39bee6dda982e9ee6d89cf481b/pkgs/applications/misc/waybar/default.nix#L123)…